src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -386,28 +405,29 @@ public static Config fromFile(final Map<String, String> configVars) throws Parse
             } else if (entry.getKey().equalsIgnoreCase("rdfLang")) {
                 c.setRdfLanguage(entry.getValue());
             } else if (entry.getKey().trim().equalsIgnoreCase("binaries")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setIncludeBinaries(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "binaries configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "binaries", lineNumber));
+            } else if (entry.getKey().trim().equalsIgnoreCase("overwriteTombstones")) {
+                c.setIncludeBinaries(parseBoolean(entry, "overwriteTombstones", lineNumber));
             } else if (entry.getKey().trim().equalsIgnoreCase("external")) {
-                if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
-                    c.setRetrieveExternal(Boolean.parseBoolean(entry.getValue()));
-                } else {
-                    throw new ParseException(String.format(
-                        "external configuration parameter only accepts \"true\" or \"false\", \"{}\" received",
-                        entry.getValue()), lineNumber);
-                }
+                c.setIncludeBinaries(parseBoolean(entry, "external", lineNumber));
             } else {
                 throw new ParseException(String.format("Unknown configuration key: {}", entry.getKey()), lineNumber);
             }
         }
         return c;
     }
 
+    private static boolean parseBoolean(final Map.Entry<String, String> entry, final String key,
+            final int lineNumber) throws ParseException {
+        if (entry.getValue().equalsIgnoreCase("true") || entry.getValue().equalsIgnoreCase("false")) {
+            return Boolean.parseBoolean(entry.getValue());
+        } else {
+            throw new ParseException(String.format(
+                "configuration parameter \"{}\" only accepts \"true\" or \"false\", \"{}\" received",
+                key, entry.getValue()), lineNumber);
```

Comment:
Note the extra close paren after `entry.getValue()` — that's the end of the `String.format()` call — the `lineNumber` argument is to the outer `ParseException` constructor.  Passing the value instead of the entry will help slim this down and make that more legible.